anyway/request_params.py

```diff
@@ -33,6 +33,7 @@ class RequestParams:
     start_time: datetime.date
     end_time: datetime.date
     lang: str
+    description: Optional[str]
```

Comment:
Hi Shay, I think it is worthwhile to name the field so it will be clear that the description comes from news flash.